<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Likemusicwi</class>
 <widget class="QWidget" name="Likemusicwi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1259</width>
    <height>898</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1259</width>
    <height>898</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1259</width>
    <height>898</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #F7F9FC;</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>340</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>340</height>
      </size>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>60</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="icon_lb">
        <property name="minimumSize">
         <size>
          <width>250</width>
          <height>250</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>250</width>
          <height>250</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">border-image: url(:/res/head_2.jpg);
  border: none;
   border-radius: 20px;
    background-color: transparent;</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_2" native="true">
        <property name="styleSheet">
         <string notr="true">QPushButton {
    border: none;
       border-radius: 8px;
      background-color: #f0f3f6;
      color: black;
   }
   QPushButton:hover {
     background-color: #e4e8ec;
    }
  QPushButton:checked {
    background-color: #fc3c4d;
    }</string>
        </property>
        <widget class="QLabel" name="label">
         <property name="geometry">
          <rect>
           <x>60</x>
           <y>50</y>
           <width>250</width>
           <height>50</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>250</width>
           <height>50</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>250</width>
           <height>50</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">color: #283248;
        font-size: 33px;
        font-family: &quot;Microsoft YaHei&quot;;
        font-weight: normal;
        background-color: transparent;
font-weight: bold;                 </string>
         </property>
         <property name="text">
          <string>我喜欢的音乐</string>
         </property>
        </widget>
        <widget class="QPushButton" name="playall_btn">
         <property name="geometry">
          <rect>
           <x>60</x>
           <y>220</y>
           <width>140</width>
           <height>55</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>11</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
	background-color: #FC3B59;
    background-image: url(:/res/viewswitch/playall1.png);  /* 默认状态图片 */
    background-repeat: no-repeat;
    background-position: left center;
}

QPushButton:hover {
background-color: #e33550;
    background-image: url(:/res/viewswitch/playall2.png);  /* 鼠标悬浮时显示的图片 */
    background-repeat: no-repeat;
    background-position: left center;
}

QPushButton:pressed {
 background-color: #eb8491;
    background-image: url(:/res/viewswitch/playall3.png);    /* 选中状态图片 */
    background-repeat: no-repeat;
    background-position: left center;
}
</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_4" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="label_6">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>280</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>280</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">border-image: url(:/res/viewswitch/kong2.png);
  border: none;
 
    background-color: transparent;</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="styleSheet">
      <string notr="true">QScrollArea {
    border: none; /* 去除边框 */
    background-color: #f0f3f6;
}

QScrollArea::corner {
    background-color: #f0f3f6;
}

QScrollBar:vertical {
    background-color: #f0f3f6; /* 滚动条背景 */
    width: 9px; /* 滚动条宽度 */
    margin: 0px 0px 0px 0px; /* 控制滚动条的间距 */
    border-radius: 4px;
}

QScrollBar::handle:vertical {
    background-color: #dcdfe4; /* 滚动条滑块颜色 */
    border-radius: 4px;
}

QScrollBar::handle:vertical:hover {
    background-color: #bec3ca; /* 滚动条滑块悬停时颜色 */
}

QScrollBar::add-line:vertical, QScrollBar::sub-line:vertical {
    background-color: #f0f3f6; /* 隐藏滚动条上下箭头 */
}
QScrollBar:horizontal {
    height: 0px; /* 关键：将高度设为 0，使水平滚动条不可见且不占空间 */
    background-color: transparent; /* 可选：背景透明 */
}
</string>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1259</width>
        <height>278</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QWidget" name="widget_3" native="true">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>50</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>50</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: #F7F9FC;</string>
         </property>
         <widget class="QLabel" name="label_2">
          <property name="geometry">
           <rect>
            <x>80</x>
            <y>0</y>
            <width>40</width>
            <height>40</height>
           </rect>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>40</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">color: #8b949e;
        font-size: 20px;
        font-family: &quot;Microsoft YaHei&quot;;
        font-weight: normal;
        background-color: transparent;
            </string>
          </property>
          <property name="text">
           <string>#</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_3">
          <property name="geometry">
           <rect>
            <x>135</x>
            <y>0</y>
            <width>60</width>
            <height>40</height>
           </rect>
          </property>
          <property name="minimumSize">
           <size>
            <width>60</width>
            <height>40</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">color: #8b949e;
        font-size: 20px;
        font-family: &quot;Microsoft YaHei&quot;;
        font-weight: normal;
        background-color: transparent;</string>
          </property>
          <property name="text">
           <string>标题</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_4">
          <property name="geometry">
           <rect>
            <x>640</x>
            <y>0</y>
            <width>60</width>
            <height>40</height>
           </rect>
          </property>
          <property name="minimumSize">
           <size>
            <width>60</width>
            <height>40</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">color: #8b949e;
        font-size: 20px;
        font-family: &quot;Microsoft YaHei&quot;;
        font-weight: normal;
        background-color: transparent;</string>
          </property>
          <property name="text">
           <string>专辑</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_5">
          <property name="geometry">
           <rect>
            <x>1110</x>
            <y>0</y>
            <width>60</width>
            <height>40</height>
           </rect>
          </property>
          <property name="minimumSize">
           <size>
            <width>60</width>
            <height>40</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">color: #8b949e;
        font-size: 20px;
        font-family: &quot;Microsoft YaHei&quot;;
        font-weight: normal;
        background-color: transparent;</string>
          </property>
          <property name="text">
           <string>时长</string>
          </property>
         </widget>
         <widget class="Line" name="line">
          <property name="geometry">
           <rect>
            <x>52</x>
            <y>49</y>
            <width>1146</width>
            <height>1</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">QFrame {
    background-color: #d2d6dc;
    height: 1px;
    border: none;
}
</string>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QListView" name="m_listView">
         <property name="styleSheet">
          <string notr="true"> border: none; /* 去除边框 */
    background-color: #F7F9FC;
</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
